### ISSUE TITLE
`hlx up` should default to the preview domain set in sidekick

### ISSUE THREAD

**Issue by davidnuescheler** [adobe/helix-cli#2219]
it would be great if we could automatically use the custom preview URLs for `hlx up`, i think this would allow devs at customers that have custom preview URLs to work more intuitively with custom CDN layers that are sitting infront of `.page`

**Comment by davidnuescheler** [comment]
thoughts @rofe / @tripodsan ?

**Comment by tripodsan** [comment]
good idea - but there is no possibility to communicate with the sidekick from the helix-cli.....
we could call `https://admin.hlx.page/sidekick/.../env.json` but this would only work for projects w/o admin authentication

**Comment by davidnuescheler** [comment]
let's add a hlx env file with default options in the github repo / root directory.

**Comment by tripodsan** [comment]
> let's add a hlx env file with default options in the github repo / root directory.

I'm not a big fan of adding even more configuration to the system. note, that devs already can add a `HLX_URL=...` in their `.env` to proxy a specific page

**Comment by tripodsan** [comment]
this is fixed.

**Comment by 3vil3mpir3** [comment]
After testing, AEM up w/out the --url flag doesn't seem to work for customer.  It does load adobe origin urls, just not the customer's worker urls

**Comment by tripodsan** [comment]
@3vil3mpir3 please note, that this doesn't work for authenticated projects. I assume that's why it doesn't work for some customers.
